client/web/src/marketing/SurveySuccess.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+import { Button, Link } from '@sourcegraph/wildcard'
+
+import { Toast } from './Toast'
+
+import styles from './SurveySuccess.module.scss'
+
+interface SurveySuccessProps {
+    handleDismiss: () => void
```

Comment:
We usually use `handle*` for handers defined in the component and `on*` for props. 

```suggestion
    onDismiss: () => void
```